keyboards/evyd13/atom47/keymaps/phsc138/readme.md

```diff
@@ -0,0 +1,63 @@
+# PHSC138's Layout for the atom47
+## Layers
+<!-- TODO: pictures -->
+```c
```

Comment:
You don't really need to duplicate the entire keymap in the readme...

Reply:
I was planning on adding some nice pictures here, finally got around to it haha